pages/consortium.html

```diff
@@ -0,0 +1,140 @@
+<#def TITLE>The Coq Consortium</#def>
+<#include "incl/header.html">
+
+<div class="framework">
+<div class="frameworklabel">A pool of resources and a platform for exchanges</div>
+<div class="frameworkcontent">
+
+<p>The objective of the Consortium is to make the community of users feel empowered with respect to
+    the future of the Coq system, giving them opportunities to see how the system will evolve, where it is
+    going, and to take an active part in this process. On the one hand, we wish to establish <strong>transparent
+    ways of contributing financially and technically to the development of the system</strong>; on the other
+    hand we wish to <strong>make sure that the most pressing needs of the user community are addressed
+    efficiently</strong> by providing a fantastic working environment to some of the most competent developers
+    trained in our teams.</p>
+
+</div>
+
+</div>
+
+<div class="framework">
+<div class="frameworklabel">Support services</div>
+<div class="frameworkcontent">
+    <p>The Consortium provides a package of support services:
+        <ul>
+            <li>Premium support for bugs. Reports by Consortium members will be processed in priority by
+the Consortium engineers. Private bug reports can be processed by Consortium engineers
+                under an NDA to explore a code base and minimize a bug example.</li>
+            <li>A session of on-site training each year. (Optional, travel and lodging expenses should be paid
+                by the member).</li>
+            <li>Integration of some code from members in regression tests.</li>
+            <li>Privileged access to Coq developers via a dedicated mailing list.</li>
+            <li>Advertisement of the member’s involvement in the Coq community.</li>
+        </ul>
+
+
+    </p>
+    <p>Partners declare a number of intensive users of Coq they represent, and the Consortium recommends a corresponding level of
+        service (a number of engineer hours, which can be used for support or training). The cost corresponds to that specific level of service. Academic members receive a 50% discount.</p>
+    <table>
+        <tr><td>Users</td><td>Engineer hours</td><td>1 year</td><td>2 years</td><td>3 years</td></tr>
+        <tr>
+            <td>
+                5-7
```

Comment:
Is it really wanted that the first line, which is per user, be restricted to 5-7 users and not to any size < 8 users?

Reply:
Yes